systemdev/kernel-pinephone.rst

```diff
@@ -0,0 +1,73 @@
+PinePhone and PineTab kernel
+============================
+
+This document describes how to build and install the PinePhone or PineTab kernel. First the recovery system is explained. Then the actual building and installation. At the bottom there are some references.
+
+Recovery
+--------
+
+The UBports image contains a `recovery <https://github.com/ubports/jumpdrive>`_ boot mode based on "jumpdrive". To get into the recovery press and hold the power and volume-up buttons until the led turns on.
```

Comment:
Sorry, but almost missed this! It's the Ubuntu Touch image, not the UBports image.

Reply:
fixed